### DISCUSSION TITLE
FontAwesome 5 | Not all icons are visible in the selection menu

### DISCUSSION THREAD

**Discussion by vysharkd** [skartknet/Iconic#32] [General]
My configuration
<img width="607" alt="thing1" src="https://user-images.githubusercontent.com/68770501/230561540-f40bcc84-5f85-4596-bd7d-77f1c1981d54.PNG">

![image](https://user-images.githubusercontent.com/68770501/230561787-8ccad8d2-76d2-4bb5-8a7c-b5c6f889cc69.png)

Issue I'm facing : 

Not all 800+ icons are visible in my icon picker in my document type.

**Comment by bm-bgtec** [comment]
I am facing the exact same problem with Live Awesome.

Most of the icons show up, but not all.

**Comment by skartknet** [comment]
Hi guys, did you manage to solve this?

**Comment by KevinHolmstrom** [comment]
I'm having the same problem. I'm using version 5.3.0 of iconic

**Comment by skartknet** [comment]
Can anyone give an update on this? Is it solved?
